cmd/pool/order.go

```diff
@@ -696,8 +696,9 @@ func parseBaseBid(ctx *cli.Context) (*poolrpc.Bid, *sidecar.Ticket, error) {
 	if ctx.IsSet("self_chan_balance") {
 		bid.SelfChanBalance = ctx.Uint64("self_chan_balance")
 		bidAmt := btcutil.Amount(bid.Details.Amt)
-		err := sidecar.CheckOfferParams(
-			bidAmt, btcutil.Amount(bid.SelfChanBalance),
+		err := order.CheckOfferParams(
```

Comment:
Do we also need to make sure here that if an order is created for a sidecar (manually) that the auction type can only be outbound liquidity? Otherwise we create an invalid order (since the sidecar itself doesn't encode the auction type).